src/includes/getting-started-verify/android.okhttp.mdx

```diff
@@ -0,0 +1,20 @@
+```kotlin
+import io.sentry.android.okhttp.SentryOkHttpInterceptor
+import okhttp3.OkHttpClient
+import okhttp3.Request
+
+private val client = OkHttpClient.Builder()
+  .addInterceptor(SentryOkHttpInterceptor())
+  .build()
```

Comment:
`l`: We already have this in the `getting-started-config`. Maybe we can ditch it here?

<img width="774" alt="Screenshot 2021-03-24 at 11 21 52" src="https://user-images.githubusercontent.com/2443292/112294232-24c67880-8c93-11eb-8976-af8c7a2b2325.png">